app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<% @rooms.each do |room| %>
+  <p>Suite: <%= room.suite %></p>
+  <p>Price: <%= room.rate %></p>
+  <p>Hotel: <%= room.hotel.name %></p>
+  <p>Number of Guests: <%= room.guests.count %></p>
```

Comment:
Doing `.count` here is an MVC infraction, since `.count` changes the SQL that the database uses to find the number of guests for this room. As such, it belongs in the model as a new method, and then we should test that method in a model test. 

If you want to try it yourself, put a `binding.pry` here and type `room.guests.count.to_sql` - see what the SQL is returned as!